src/components/Auth/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent, FocusEventHandler } from "react";
+import styles from "./Input.module.scss";
+
+interface Input {
+  className: string;
+  type: string;
+  name: string;
+  value: any;
```

Comment:
타입을 조금 더 좁힐 수 있을 것 같은데요~
이정도로 표현해주면 좋을 것 같습니다~
```suggestion
  value: string | number;
```